app/controllers/triannon/auth_controller.rb

```diff
@@ -5,64 +5,56 @@ module Triannon
   class AuthController < ApplicationController
     include RdfResponseFormats
 
-    AUTH_EXPIRY  =   60 # seconds
-    TOKEN_EXPIRY = 3600 # seconds
+    # HTTP request methods accepted by /auth/login
+    LOGIN_ACCEPT = 'OPTIONS, GET, POST'
 
     # OPTIONS /auth/login
     def options
```

Comment:
I looked, lackadaisically, for the spec about this options request - I didn't find it.  Is it part of IIIF auth spec, or Rob's spec, or??